core/services/monitoring/client.go

```diff
@@ -0,0 +1,101 @@
+package monitoring
+
+import (
+	"context"
+	"sync/atomic"
+
+	"github.com/smartcontractkit/libocr/commontypes"
+	"github.com/smartcontractkit/wsrpc"
+)
+
+type Client interface {
+	commontypes.MonitoringEndpoint
+
+	Start()
```

Comment:
Please use either Start/Stop or Open/Close for lifecycle functions.

I suggest we use:

- Open/Close for clients: a connection you either open yourself or one gets an already connected client and just uses it
- Start/Stop for services: components/services that are started/stopped by the system (as one of root subsystems) or by other services that manage multiple subservices.

So in this case I'd suggest we design a TelemetryService that we start on system start. That service onStart opens a TelemetryClient connection and exposes it as a port (interface others can inject and use directly).